packages/main/cypress/specs/ColorPalette.cy.tsx

```diff
@@ -44,4 +61,150 @@ describe("Color Palette tests", () => {
+			<ColorPalette id="cp4" showMoreColors showRecentColors>
+				<ColorPaletteItem value="pink"></ColorPaletteItem>
+				<ColorPaletteItem value="darkblue"></ColorPaletteItem>
+				<ColorPaletteItem value="#444444"></ColorPaletteItem>
+				<ColorPaletteItem value="rgb(0,200,0)"></ColorPaletteItem>
+				<ColorPaletteItem value="green"></ColorPaletteItem>
+				<ColorPaletteItem value="darkred"></ColorPaletteItem>
+				<ColorPaletteItem value="yellow"></ColorPaletteItem>
+				<ColorPaletteItem value="blue"></ColorPaletteItem>
+				<ColorPaletteItem value="cyan"></ColorPaletteItem>
+				<ColorPaletteItem value="orange"></ColorPaletteItem>
+				<ColorPaletteItem value="#5480e7"></ColorPaletteItem>
+				<ColorPaletteItem value="#ff6699"></ColorPaletteItem>
+			</ColorPalette>
+		);
+
+		// click on first 5 color palette items
+		cy.get("#cp4")
+			.find("ui5-color-palette-item")
+			.then(items => {
+				for (let i = 0; i < 5; i++) {
+					cy.wrap(items[i]).click();
+				}
+			});
+
+		// check recent colors
+		cy.get("#cp4")
+			.shadow()
+			.find(".ui5-cp-recent-colors-wrapper ui5-color-palette-item")
+			.should("have.length", 5)
+			.then(items => {
+				cy.wrap(items[0]).should("have.value", "green");
+				cy.wrap(items[1]).should("have.value", "rgb(0,200,0)");
+				cy.wrap(items[2]).should("have.value", "#444444");
+				cy.wrap(items[3]).should("have.value", "darkblue");
+				cy.wrap(items[4]).should("have.value", "pink");
+			});
+	});
+
+	it("Tests if only one item is selected at a time in the color palette", () => {
+		cy.mount(
+			<ColorPalette id="cp1SelectedTest">
+				<ColorPaletteItem value="darkblue" selected></ColorPaletteItem>
+				<ColorPaletteItem value="pink" selected></ColorPaletteItem>
+				<ColorPaletteItem value="#444444" selected></ColorPaletteItem>
+				<ColorPaletteItem value="rgb(0,200,0)" selected></ColorPaletteItem>
+				<ColorPaletteItem value="green"></ColorPaletteItem>
+				<ColorPaletteItem value="darkred"></ColorPaletteItem>
+				<ColorPaletteItem value="yellow"></ColorPaletteItem>
+				<ColorPaletteItem value="blue"></ColorPaletteItem>
+				<ColorPaletteItem value="cyan"></ColorPaletteItem>
+				<ColorPaletteItem value="orange"></ColorPaletteItem>
+				<ColorPaletteItem value="#5480e7"></ColorPaletteItem>
+				<ColorPaletteItem value="#ff6699"></ColorPaletteItem>
+			</ColorPalette>
+		);
+
+		cy.get("#cp1SelectedTest")
+			.find("ui5-color-palette-item")
+			.eq(0)
+			.click();
+
+		cy.get("#cp1SelectedTest")
+			.find("ui5-color-palette-item")
+			.eq(1)
+			.click();
+
+		cy.get("#cp1SelectedTest")
+			.find("ui5-color-palette-item")
+			.eq(0)
+			.should("not.have.attr", "selected");
+
+		cy.get("#cp1SelectedTest")
+			.find("ui5-color-palette-item")
+			.eq(1)
+			.should("have.attr", "selected");
+	});
+
+	it("Tests if clicking on selected item, does not deselect it", () => {
+		cy.mount(
+			<ColorPalette id="cp1SelectedTest">
+				<ColorPaletteItem value="darkblue" selected></ColorPaletteItem>
```

Comment:
We recommend setting boolean props, like `selected`, to true explicitly: selected={true}